.github/ISSUE_TEMPLATE/bug-desktop.yml

```diff
@@ -1,7 +1,6 @@
 name: Bug report for the Element desktop app (not in a browser)
 description: File a bug report if you are using the desktop Element application.
-title: "[Bug]: "
-labels: [T-Defect, triage]
+labels: [T-Defect]
 assignees:
   -   
```

Comment:
While you're at it, is this necessary?

Reply:
We don't at the moment, but we could potentially assign it to a triage team in the first instance to bring it into a queue. I'm getting feedback from Germain (and hopefully palid this afternoon) on how they did today before proposing something. Can take it out for now and add it back if needed later.